seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/multitablesink/MultiTableSinkWriter.java

```diff
@@ -267,26 +276,32 @@ public void abortPrepare() {
 
     @Override
     public void close() throws IOException {
-        Throwable firstE = null;
+        final Throwable[] firstE = {null};
```

Comment:
Add comment for why use array.

Reply:
> I get it. I think you should add a comment for it

Ok, I have added it and am waiting to submit it along with other questions